_posts/2023-07-20-bazel-q2-2023-community-update.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: posts
+title: "Bazel Q2 2023 Community Update"
+authors:
+ - keertk
+ - radvani
+---
+
+## Product updates
+
+### Feature improvements
+
+**Bisect feature in Bazelisk:** The `--bisect` flag was introduced in [Bazelisk v1.17.0](https://github.com/bazelbuild/bazelisk/releases/tag/v1.17.0). It allows you to bisect Bazel versions to find out the specific commit that caused a build failure. You can find more information and examples [here](https://github.com/bazelbuild/bazelisk#--bisect).
+
+**Lock file for Bzlmod:** Preliminary support for a lockfile for Bzlmod was introduced in [Bazel 6.2.0](https://github.com/bazelbuild/bazel/releases/tag/6.2.0). It's disabled by default; use [--lockfile_mode](https://bazel.build/reference/command-line-reference#flag--lockfile_mode)=update to enable it. The lockfile contains only Bazel module information; it does not involve module extensions yet.
```

Comment:
nit: "Lockfile" instead of "Lock file" for consistency

Reply:
Fixed